src/usgs_stations_json_csv.py

```diff
@@ -47,7 +47,7 @@ def read_usgs_json(json_path):
         # Iterate over different columns. Each colum can be a component
         data = []
         pgas = []
-        for index, chan in rec_sation.iteritems():
+        for index, chan in rec_sation.items():
```

Comment:
1. `rec_sation` → `rec_station`? 

2. if `index` is not used anywhere later in the loop, consider replacing with `_` (python convention)